assignment/scripts/3-music-collection.js

```diff
@@ -1 +1,123 @@
 console.log('***** Music Collection *****')
+
+
+let collection = [];
+
+function addToCollection(title, artist, yearPublished, tracks){
+    let newAlbum = {
+        title: title ,
+        artist: artist ,
+        yearPublished: yearPublished,
+        tracks: tracks,
+    }
+    collection.push(newAlbum);
+    return collection;
```

Comment:
Whoops! The requirements specify that the new album should be returned, not the collection.